### ISSUE TITLE
Not an unenrollment method

### ISSUE THREAD

**Issue by JackWagon885** [theskid22/WHIMMER#1]
So not only is this a bit poorly worded, it's also NOT an unenrollment method.

Instead, this a guide on how to boot from USB in Windows 10/11, & a very poor set of instructions on creating multiple partitions.

Speaking of, just creating multiple partitions does nothing. You can install multiple OS's on 1 USB device via Ventoy though. While this is useful, creating a hidden partition is better in this situation.

The ideal version of this is a USB device with the dummy OS being the one your school computer uses, then having a hidden OS (or more) for personal use.

One important detail you forgot is the fact that in order to save things to the USB device, you need to create a live bootable instance with persistent memory.

In summary, this is just a poor guide on a well known feature that is useful for bypassing.

One last thing, if you want to do this on ChromeOS, you have to:
-Get the school's WiFi password.
-Enter Recovery Mode (Esc+Refresh+Power)
-Enable Dev Mode (Ctrl+D)
-This will powerwash your computer & can be blocked.
-Press Ctrl+Alt+F2 shortly after booting
-Enter a command
-Restart the computer
-Press Ctrl+U.
